tests/unit/lms/services/application_instance_getter_test.py

```diff
@@ -69,40 +69,38 @@ def test_provisioning_returns_False_if_consumer_key_unknown(self, ai_getter):
         assert not ai_getter.provisioning_enabled()
 
     @pytest.mark.parametrize(
-        "section_groups_feature_flag,developer_key,canvas_sections_enabled,expected_result",
+        "params",
         [
-            (True, "test_developer_key", True, True),
-            (False, "test_developer_key", True, False),
-            (True, None, True, False),
-            (True, "test_developer_key", False, False),
-            (False, None, False, False),
+            dict(),
+            dict(feature_flag=False, expected_result=False),
+            dict(developer_key=None, expected_result=False),
+            dict(canvas_sections_enabled=False, expected_result=False),
```

Comment:
What's the difference between this and a dict literal? 

Weird aside: I was looking it up and apparently literals are 2x the speed. Not relevant here, but interesting anyway.

Reply:
As `expected_result` is mandatory it might be nice to keep that outside of the dict so we have an input output divide:

```python
({}, True),
({"feature_flag": False}, False),
({"developer_key": None}, False),
...
```